src/types/files.ts

```diff
@@ -1,13 +1,20 @@
 import { Image } from "./image";
 
+type FilePath = string;
```

Comment:
```suggestion
type FilePath = string & {}
```
to avoid the resoliton of the alias ?

Reply:
VS code displays it un-resolved for me :man_shrugging: 
`(method) FileStore.upload(file: File): Promise<FilePath>`

Reply:
indeed does not work